relay-dynamic-config/src/feature.rs

```diff
@@ -99,6 +99,13 @@ pub enum Feature {
     #[serde(rename = "organizations:indexed-spans-extraction")]
     ExtractSpansFromEvent,
 
+    /// Enables description scrubbing for MongoDB spans (and consequently, their presence in the
+    /// Queries module inside Sentry).
+    ///
+    /// Serialized as `organizations:performance-queries-mongodb-extraction`.
+    #[serde(rename = "organizations:performance-queries-mongodb-extraction")]
+    ScrubMongoDBDescriptions,
```

Comment:
nit: 
```suggestion
    ScrubMongoDbDescriptions,
```

https://rust-lang.github.io/api-guidelines/naming.html#:~:text=In%20UpperCamelCase%2C%20acronyms%20and%20contractions%20of%20compound%20words%20count%20as%20one%20word%3A%20use%20Uuid%20rather%20than%20UUID